Sources/HTMLEncoding.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HTMLEncoding.swift
+//  Diagnostics
+//
+//  Created by David Steppenbeck on 2020/02/26.
+//  Copyright © 2020 WeTransfer. All rights reserved.
+//
+
+import Foundation
+
+public protocol HTMLEncoding {
+    func addingHTMLEncoding() -> HTML
+}
+
+extension String: HTMLEncoding {
+    
+    /// Encodes entities to be displayed correctly in the final HTML report.
+    public func addingHTMLEncoding() -> HTML {
+        return self.replacingOccurrences(of: "<", with: "&lt;")
```

Comment:
```suggestion
        return replacingOccurrences(of: "<", with: "&lt;")
```

`self` is not needed here